libnmstate/nm/applier.py

```diff
@@ -390,6 +391,11 @@ def _build_connection_profile(iface_desired_state, base_con_profile=None):
             )
             settings.append(linux_bridge_setting)
     elif iface_type == ovs.BRIDGE_TYPE:
+        for port in iface_desired_state.get(OvsB.CONFIG_SUBTREE, {}).get(
+            OvsB.PORT_SUBTREE, []
+        ):
+            if OvsB.Port.LINK_AGGREGATION_SUBTREE in port:
+                raise NmstateNotImplementedError
```

Comment:
This is not the place we raise the missing implementation.
Missing implementations are usually raised in the setting creation.
For example: https://github.com/nmstate/nmstate/blob/95af61535301ee885d415f5853aa2c06c23972ad/libnmstate/nm/bridge.py#L93